modules/administration-guide/attachments/restricted-environment/prepare-restricted-environment.sh

```diff
@@ -93,7 +99,7 @@ spec:
   image: $my_operator_index
 EOF
 
-# Create CheCluster custom resource configuration file
+echo "Creating the 'che-operator-cr-patch.yaml' CheCluster custom resource configuration file locally."
```

Comment:
This is a mouthful. It's very specific but the vaguer "Creating the 'che-operator-cr-patch.yaml' file locally." or "Creating CheCluster custom resource configuration file locally." is better suited as a title imho.

Reply:
done